script/common/deployment/01_deployToken.s.sol

```diff
@@ -31,13 +36,29 @@ contract DeployToken is Script {
         // deploy Token facet
         Token tokenFacet = new Token();
 
+        uint256 canDeployTokenBridgeFacet = vm.envUint(
```

Comment:
Since you asked me to write also the script for cutting the bridge facet I imagined you would have needed a scenario where the bridge facet could have been not deployed at immediately. In any fixed here https://github.com/giovannifranchi/insrt-v2-contracts/commit/8bcd226fcc282b73e1ffb2409fc5bfeb8eaf3bdd